### PR TITLE
Add color formatting for ANSI chars in logs from task executions

### PR DESCRIPTION
**Pull request by jscheffl**
We use a lot of colored shell scripts in our environment which always produce ugly log displays in UI... that was bothering me a while already.

This PR adds color to log display on UI. Color brings life into IT and makes me - and hopefully more people happy.

What changed:
- Added some color to the `example_bash_operator` DAG
- Added coloring for GRID view log display
- Added coloring for "More logs" view

Grid log view BEFORE this PR:
image

More log view BEFORE this PR:
image

Grid log view AFTER this PR:
image

More log view AFTER this PR:
image

FYI @wolfdn @clellmann @AutomationDev85

### CODE REVIEW

**Review by bbovenzi — APPROVED** [review]
Love it!

**Review by potiuk — APPROVED** [review]
Beautiful and simple !

**Comment by bbovenzi** [comment]
To fix the CI:

Can you add the package `"ansi_up"` to the  `transformIgnorePatterns` list in  `jest.config.js`?

**Comment by jscheffl** [comment]
I just realized that the "new" integrated lib also can take care of HTML escaping and URLifying, that will be an option to clean custom code.

**Comment by potiuk** [comment]
> I just realized that the "new" integrated lib also can take care of HTML escaping and URLifying, that will be an option to clean custom code.

Oh wow. Even better :)

**Review by potiuk — APPROVED** [review]

**Review by Taragolis — APPROVED** [review]
awesome

**Review by dirrao — APPROVED** [review]

**Comment by jscheffl** [comment]
Oh, wow 12 approvals in <12h :-D
I found one glitch with the re-work, please do not merge (yet), need to fix this in the afternoon so that the linkify is working again.

File: airflow/www/jest.config.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ const config = {
   transformIgnorePatterns: [
     `node_modules/(?!${[
       // specify modules that needs to be transformed for jest. (esm modules)
+      "ansi_up",
```
Comment (potiuk) [inline airflow/www/jest.config.js:35]: Just opened conversation to keep it non-mergable until @jscheffl  fixes the glitch

**Review by eladkal — APPROVED** [review]

**Comment by jscheffl** [comment]
Sorry that I needed re-work a bit - I needed to realize that the `ansi_up` does only handle the URLyfication only for ANSI encoded URLs, not plain text URLs. So needed to roll-back a bit - and then stumbeld about a double-tag-sanititization - cleaned this redundancy.

Now this nice features makes the codebase 27 lines smaller by adding this cool feature :-D

I'd wait for merge until I get a fresh review as code has been changed a lot since previous reviews...

**Review by potiuk — APPROVED** [review]

**Pull request merged**